ansible_bender/db.py

```diff
@@ -83,16 +83,15 @@ def acquire(self):
         """
         while True:
             try:
-                with open(self._lock_path(), "r") as fd:
+                with os.fdopen(os.open(self._lock_path(), os.O_RDONLY ),'r') as fd:
```

Comment:
If the file exists then it opens with the reading operation to get the process ids